app/main.py

```diff
@@ -1 +1,25 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    if not command.startswith("mv "):
+        raise ValueError(
+            "Command must start with 'mv '."
+            "")
```

Comment:
The empty string in line 8 is unnecessary and can be removed. It doesn't affect functionality but may cause confusion.